DESCRIPTION

```diff
@@ -26,6 +26,7 @@ Depends:
 LazyData: true
 Imports: 
     dplyr,
+    GAMBLR.helpers,
```

Comment:
Do we really need `GAMBLR.helpers` here? I think there is no `@import GAMBLR.helpers` tag in any function documentation.

Reply:
Let me try if it can be omitted from the DESCRIPTION.